tests/Kron.py

```diff
@@ -37,14 +39,111 @@
 
 #-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-
 
-class KronPhotometryTestCase(unittest.TestCase):
+def makeGalaxy(width, height, flux, a, b, theta, dx=0.0, dy=0.0, xy0=None, xcen=None, ycen=None):
```

Comment:
Thanks Perry!

One thing, though: I am *definitely not* saying you should merge to master before asking for review. I'm suggesting that you should at least consider *rebasing* your branch against the latest master. That makes it easier for the reviewer to see how your changes will integrate with the rest of the codebase, and it will make it much easier to merge the code after the review is complete. Plus, in this case, it would have quickly exposed the issue with `makeGalaxy`.

As far as I'm aware, though, there's no *requirement* in our procedures to rebase against master before the review; it's just something that I'm suggesting you should consider.